src/dotnet/SlnFileExtensions.cs

```diff
@@ -28,66 +27,68 @@ public static void AddProject(this SlnFile slnFile, string fullProjectPath)
                 fullProjectPath);
 
             if (slnFile.Projects.Any((p) =>
-                    string.Equals(p.FilePath, relativeProjectPath, StringComparison.OrdinalIgnoreCase)))
+                string.Equals(p.FilePath, relativeProjectPath, StringComparison.OrdinalIgnoreCase)))
             {
                 Reporter.Output.WriteLine(string.Format(
                     CommonLocalizableStrings.SolutionAlreadyContainsProject,
                     slnFile.FullPath,
                     relativeProjectPath));
+                return;
```

Comment:
I really, really appreciate the contribution, but there appears to be some stylistic changes here that are not part of the stated purpose of the PR - in this case, an early return. We need to avoid this to keep the risk low and simplify PR review. This return changes code flow, so the reviewers have to worry about whether the entire method is shown in the review window. And every line of code that is changed (even whitespace) adds to the overhead.

Reply:
Hi @KathleenDollard. Thanks for the review! In my opinion it would have made the code a bit more readable. But I understand that it's a lot simpler for the reviewer, if the PR doesn't include a lot of stylistic changes like that. I reverted the stylistic changes (which were mostly in `src/dotnet/SlnFileExtensions.cs`) in 
8b7b6ca.